test/verify/check-system-realms

```diff
@@ -1160,16 +1160,20 @@ class TestPackageInstall(packagelib.PackageCase):
         # case 1: disable PackageKit
         m.execute("systemctl mask packagekit; systemctl stop packagekit.service || true")
         self.login_and_go("/system")
-        b.wait_text(self.domain_sel, "Join domain")
+        b.reload()
+        b.enter_page("/system")
+        b.wait_text(self.domain_sel, "Install realmd support")
```

Comment:
This can't work -- this case tests disabling packagekit, so `packagekit.detect()` will be false. As we don't have a state for "realmd does not exist and we can't install it", keeping the disabled "Join domain" is fine for now.